Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ghcr.io/medibloc/panacea-core:master
```

Comment:
Do we need to github auth token to fetch this docker image? I've never checked it so far.

If we need auth tokens, I think we can move to Docker Hub.
I've checked the pricing policy of Docker Hub quickly. It seems that it's free to publish unlimited docker images if our git repo is public :).

Reply:
Yes. It need to log in to ghrc.io
As you suggested, It would be good to move to Docker Hub.